src/raw.js

```diff
@@ -136,6 +140,10 @@ function replaceKeyBindings(raw) {
     return full.replace(key, '?')
   })
 
+  emptyBindings.map(function(binding) {
+    sql = sql.replace(new RegExp(` ${binding}|${binding} |${binding}`, 'g'), '');
```

Comment:
The previous regular expression matched by forcing a trailing space, so it would convert the following:

`SELECT :col`
to
`SELECT`

Whereas removing the space requirement from the regular expression would instead convert

`SELECT :col`
to
`SELECT ` *(Space remains)*

Generally this shouldn't be an issue execution wise, but from the failing tests I figured why not keep it that way. Basically if a binding is supplied in the query like `:col`, but the binding value is undefined, the previous could would remove that binding from the query and its trailing space. That's what I'm doing in that code to supplement the removal of space from the regex.